woocommerce/woocommerce
# Remove feature config from repo and add it to the gitignore

Fixes: #32306 

### Changes proposed in this Pull Request:

Cleanup from the WooCommerce Admin merge, this adds the feature-config back to the `.gitignore` and adds the build `feature-config` script to the PHP tests so they run successfully.

### How to test the changes in this Pull Request:

1. Check if all actions pass
2. Load the PR and run `pnpm nx build woocommerce` and see if it still generates the `feature-config`
